packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -18,7 +18,7 @@ import {Texture as ThreeTexture, Vector2} from 'three';
 import {Filter, MagFilter, MinFilter, Wrap, WrapMode} from '../../three-components/gltf-instance/gltf-2.0.js';
 import {Sampler as DefaultedSampler} from '../../three-components/gltf-instance/gltf-defaulted.js';
 
-import {Sampler as SamplerInterface} from './api.js';
+import {Sampler as SamplerInterface, Vector2 as Vector2Interface} from './api.js';
```

Comment:
Maybe we should just rename `Vector2` to avoid this confusion with three.js? Also, we have a `Vector2D` and `Vector3D` in model-viewer-base.js - can we be consistent with those (accept them as valid input)?

Reply:
I initially tried to use Vector2D from model-viewer-base in the api.js file and it wouldn't work. I can give it another go at trying out deleting the api.js Vector2 interface.

`rename Vector2 to avoid this confusion with three.js? `
You mean rename the Vector2 in api.js?

Reply:
I think the interface is necessary so that users don't have to include a `toString()`. And yes, I mean renaming in api.js. Also check the members for compatibility with `Vector2D`.

Reply:
Yeah Vector2D from base.js requires additional imports. Just uploaded a new commit